lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -314,9 +314,7 @@ class SPIRVInstTemplateBase : public SPIRVInstruction {
     return Operands;
   }
 
-  virtual SPIRVValue *getOperand(unsigned I) {
-    return getOpValue(I);
-  }
+  virtual SPIRVValue *getOperand(unsigned I) { return getOpValue(I); }
```

Comment:
better not to add non-related changes to the patch

Reply:
agree, this and following are clang-formats artifacts, I'll revert to the previous state